src/streaming-client.ts

```diff
@@ -47,9 +48,13 @@ export class RevAiStreamingClient extends EventEmitter {
         this.accessToken = accessToken;
         this.config = config;
         this.baseUrl = `wss://api.rev.ai/speechtotext/${version}/stream`;
-        this.requests = new PassThrough();
+        this.requests = new PassThrough({ objectMode: true });
```

Comment:
in object mode the buffering works in a [different ](https://nodejs.org/api/stream.html#object-mode) way. so the buffering threshold works in a different way: `highWaterMark` option param sets the count obj objects to buffer rather than the buffer size in bytes.
And [I don't](https://github.com/nodejs/node/blob/master/lib/internal/streams/state.js#L11) see the default buffering value by default.
Is it correct default setup?

Reply:
I think default is fine